MekHQ/src/mekhq/campaign/universe/Planet.java

```diff
@@ -448,24 +406,37 @@ public Double getDayLength() {
         return dayLength;
     }
     
-    public Integer getSystemPosition() {
-        return sysPos;
+    public Double getYearLength() {
+        return yearLength;
     }
     
-    public String getSystemPositionText() {
-        return null != sysPos ? sysPos.toString() : "?"; //$NON-NLS-1$
+    public String getPlanetType() {
+        return planetType;
     }
-
-    public Double getOrbitEccentricity() {
-        return orbitEccentricity;
-    }
-
-    public Double getOrbitInclination() {
-        return orbitInclination;
+    
+    public Integer getSystemPosition() {
+        return sysPos;
     }
-
-    public Double getTilt() {
-        return tilt;
+    
+    /**
+     * This function returns a system position for the planet that does not account for asteroid belts. Therefore
+     * this result may be different than that actual sysPos variable.
+     * @return String of system position after removing asteroid belts
+     */
+    public String getDiplayableSystemPosition() {
+    	//We won't give the actual system position here, because we don't want asteroid belts to count
+    	//for system position
+    	if(null == getParentSystem()) {
+    		return null != sysPos ? sysPos.toString() : "?"; //$NON-NLS-1$
+    	}
+    	Integer pos = 0;
+    	for(int i = 1; i <= sysPos; i++) {
+    		//if(getParentSystem().getPlanet(i).getPlanetType().equals("Asteroid Belt")) {
```

Comment:
You've got some commented out code here that seems to match the method's description, but isn't there a planet type enum or something?